ui/pages/settings/settings.component.js

```diff
@@ -41,12 +41,32 @@ class SettingsPage extends PureComponent {
     initialBreadCrumbKey: PropTypes.string,
     mostRecentOverviewPage: PropTypes.string.isRequired,
     addNewNetwork: PropTypes.bool,
+    conversionDate: PropTypes.number,
   };
 
   static contextTypes = {
     t: PropTypes.func,
   };
 
+  state = {
+    lastFetchedConversionDate: null,
+  };
+
+  componentDidMount() {
+    this.handleConversionDate();
+  }
+
+  componentDidUpdate() {
+    this.handleConversionDate();
+  }
+
+  handleConversionDate() {
```

Comment:
Does this have to happen in this component? Could it just be in `setttings-tab.container.js`?

Reply:
oh, I see, we are using client side state to cache the `lastFetchedConversionDate`

Reply:
Hello @danjm ,

You are totally right, my first implementation was directly in setttings-tab.container.js. I did it in settings because i wanted the `date` state to be available in all settings tabs. 
Like i said in my above explanations, - in offline mode -  if we leave _settings_  and come back, date state will be lost & message displayed.  Thats normal because `date` is obviously not in appstate. I dont know if we want to put it there.

Let me know which implementation is most relevant. Thx